### PR TITLE
docs: updating major upgrade path

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

n/a

## Description of changes being made
Updating some bits so it's more sytlistically like ours, fixing linting errors, some other minor version bumps for the major upgrade path topic

## Checklist

- [x] Test all commands and procedures, if applicable.
- [x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md [hunk 1]
```diff
@@ -28,23 +28,23 @@ To successfully adapt your applications, you must have:
 Check for the following on your existing `cluster.yaml`:
 
 - One or more of `spec.kubernetes.networking.noProxy`, `spec.kubernetes.networking.httpProxy` or `spec.kubernetes.networking.httpsProxy` is set in `ClusterConfiguration`.
-- Enabled `gatekeeper` with custom `values` field in `ClusterConfiguration` in `cluster.yaml`.
+    - Enabled `gatekeeper` with custom `values` field in `ClusterConfiguration` in `cluster.yaml`.
```
Comment (vacas5) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:31]: is the leading tab intentional?

**Review by vacas5 — COMMENTED** [review]
nice finds

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md [hunk 2]
```diff
@@ -55,29 +55,29 @@ Elasticsearch, elasticsearch-curator, and Kibana Addons are deleted without unin
...
 
 You must back up any custom Grafana dashboards before running the adaption process.
 
 ## Velero
 
-In 2.x, the Velero install has moved from the `velero` namespace to the `kommander` namespace.   If you have runbooks or automation that invokes the velero tool, make sure to update them as suggested below.
+In 2.x, the Velero install has moved from the `velero` namespace to the `kommander` namespace. If you have runbooks or automation that invokes the velero tool, make sure to update them as suggested below.
 
-<p class="message--note"><strong>NOTE: </strong> The Velero namespace can be set to kommander with the <code>velero client config set namespace=kommander</code>.  Alternatively you can set the environment variable <code>export VELERO_NAMESPACE=kommander</code>, or use the velero --namespace command line option.</p>
+<p class="message--note"><strong>NOTE: </strong> The Velero namespace can be set to <code>kommander</code> with the <code>velero client config set namespace=kommander</code>.  Alternatively you can set the environment variable <code>export VELERO_NAMESPACE=kommander</code>, or use the velero --namespace command line option.</p>
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:76]:
```suggestion
<p class="message--note"><strong>NOTE: </strong> The Velero namespace can be set to <code>kommander</code> with the command <code>velero client config set namespace=kommander</code>. Alternatively you can set the environment variable <code>export VELERO_NAMESPACE=kommander</code>, or use the velero <code>--namespace kommander</code> command line option.</p>
```

**Review by gracedo — APPROVED** [review]

File: .github/styles/Vocab/Docs/accept.txt [hunk 3]
```diff
@@ -117,13 +122,15 @@ Snapshotter
...
 subcommand
+(subnet|subnets|Subnets)
 subtree
 TensorFlow
 (thanos|Thanos)
 Toree
 (traefik|Traefik)
 uncordon(|ed)
 unmount(|s|ing|ed)
+unpauses
```
Comment (cmastr) [inline .github/styles/Vocab/Docs/accept.txt:133]: why do we need this? not really even a word.

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md [hunk 4]
```diff
@@ -28,23 +28,23 @@ To successfully adapt your applications, you must have:
...
 
 If none of the conditions apply to your cluster, then you can skip to next section.
 
-1.  Because Kommander 2.0+ uses flux to manage applications, we need to configure the gatekeeper `mutatingwebhookconfigurations` (which is a cluster-scoped resource) such that it allows `dry-run` calls, which are required by the flux kustomize controller to calculate the diff of a resource. In order to do this:
+1.  Because Kommander 2.0+ uses Flux to manage applications, we need to configure the Gatekeeper `mutatingwebhookconfigurations` (which is a cluster-scoped resource) such that it allows `dry-run` calls, which are required by the Flux kustomize controller to calculate the difference of a resource. In order to do this:
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:35]:
```suggestion
1.  Because Kommander 2.0+ uses Flux to manage applications, you must configure the Gatekeeper `mutatingwebhookconfigurations` (which is a cluster-scoped resource) to allow `dry-run` calls. They are required by the Flux kustomize controller to calculate the difference of a resource. To do this:
```

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md [hunk 5]
```diff
@@ -73,13 +73,13 @@ If none of the conditions apply to your cluster, then you can skip to next secti
...
 
-    You need to copy the above configuration into `Addon` resource of `gatekeeper`. Start by printing the current `values` section:
+    You need to copy the above configuration into the `Addon` resource of `gatekeeper`. Start by printing the current `values` section:
 
     ```bash
     kubectl get addon gatekeeper -nkubeaddons -o=jsonpath={.spec.chartReference.values}
     ```
 
-    which should print an output more or less similar to the following:
+    This will print an output similar to the following:
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:82]:
```suggestion
    This will print the following output:
```

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md [hunk 6]
```diff
@@ -17,13 +17,13 @@ beta: false
 
 #### Alerting
 
-Custom `alertmanager` [configurations](https://docs.d2iq.com/dkp/konvoy/1.8/monitoring/#notify-prometheus-alerts-in-slack) must be copied or re-created [manually](https://docs.d2iq.com/dkp/kommander/2.0/monitoring/#notify-prometheus-alerts-in-slack). You can do this before adapting Prometheus, so that there is no time interval when the alertmanager is not configured.
+Custom `alertmanager` [configurations](/dkp/konvoy/1.8/monitoring/#notify-prometheus-alerts-in-slack) must be copied or re-created [manually](/dkp/kommander/2.1/monitoring/#notify-prometheus-alerts-in-slack). You can do this before adapting Prometheus, so that there is no time interval when the `alertmanager` is not configured.
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:20]:
```suggestion
You must copy or [recreate](/dkp/kommander/2.1/monitoring/#notify-prometheus-alerts-in-slack) custom `alertmanager` [configurations](/dkp/konvoy/1.8/monitoring/#notify-prometheus-alerts-in-slack) before adapting Prometheus, so that there is no time interval when the `alertmanager` is not configured.
```

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md [hunk 7]
```diff
@@ -17,13 +17,13 @@ beta: false
...
+You should copy or re-create ServiceMonitors using the label `release: kube-prometheus-stack` instead of `release: prometheus-kubeaddons`. This can be done before processing, using different names or namespaces for the new objects. Alternatively, you could just re-label any existing ServiceMonitors before or after the processing, but this will introduce an interval where Prometheus is running but the ServiceMonitors have no effect. After the Prometheus-operator in the adapted Prometheus starts, it only looks at ServiceMonitor objects with `release: kube-prometheus-stack` label, and it ignores ServiceMonitors with the old label until they are re-labeled.
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:24]:
```suggestion
Copy or re-create ServiceMonitors using the label `release: kube-prometheus-stack` instead of `release: prometheus-kubeaddons` before processing, using different names or namespaces for the new objects. Or, you can re-label any existing ServiceMonitors before or after processing, but this will introduce an interval where Prometheus is running but the ServiceMonitors have no effect. After the Prometheus-operator in the adapted Prometheus starts, it only looks at ServiceMonitor objects with `release: kube-prometheus-stack` label, and it ignores ServiceMonitors with the old label until they are re-labeled.
```

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md [hunk 8]
```diff
@@ -17,13 +17,13 @@ beta: false
...
 
-There are two ways to adapt ServiceMonitors created using Helm values of the Addon:
+You can use two ways to adapt ServiceMonitors created using Helm values of the Addon:
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:26]: Let's put this back please. There are is better than 'you can use'.

Comment (cmastr) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:26]:
```suggestion
There are two ways to adapt ServiceMonitors created using Helm values of the Addon:
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md [hunk 9]
```diff
@@ -24,7 +24,7 @@ If `keepalived` is enabled, update Konvoy 1.8 Configuration with the interface u
     ens192
     ```
 
-1.  Update `cluster.yaml` with the output of the previous commanddocs/site/major-version-upgrade/upgrade/prepare-local-environment/index.md:90
+1.  Update `cluster.yaml` with the output of the previous command:
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md:27]: I think this is supposed to be a link to.... somewhere.

**Review by cmastr — APPROVED** [review]

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md [hunk 10]
```diff
@@ -312,7 +312,7 @@ pod "capi-controller-manager-d4b9c7c4c-hkqfl" deleted
 
 ## Prepare the Dex Addon for Kubernetes v1.21.6
 
-The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as well as with v1.20.11.
+The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as with v1.20.11.
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md:315]: I think the previous wording was better here, as after this change the Dex Addon works with both Kubernetes versions.   If you think this is too wordy, perhaps something like "works correctly with both Kubernetes 1.21.6 and 1.20.11."

```suggestion
The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as well as with v1.20.11.
```

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:31]: Def not - thanks!

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md [hunk 11]
```diff
@@ -19,7 +19,7 @@ This section automatically adapts your Konvoy addons to Kommander platform appli
 
 To successfully adapt your applications, you must have:
 
--   A Konvoy 1.8.3 cluster that has already been [upgraded to DKP 2.1](https://docs.d2iq.com/dkp/konvoy/2.1/major-version-upgrade/), with the kommander addon disabled in your cluster.yaml.
+-   A Konvoy 1.8.3 cluster that has already been [upgraded to DKP 2.1](/dkp/konvoy/2.1/major-version-upgrade/), with the `kommander` addon disabled in your cluster.yaml.
```
[the remark this answers is not in the document]
Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:22]: Right - so, that was a needle I was trying to thread - when to address all versions - I'll add here now, though, that's a good catch.

File: .github/styles/Vocab/Docs/accept.txt (hunk 3 above)
Comment (msdolbey) [reply .github/styles/Vocab/Docs/accept.txt:133]: lol - I guess 'pause' is a specific thing for cluster objects, but I'll reword this. It just got caught because I edited the same page that had this word in there.

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md [hunk 12]
```diff
@@ -53,7 +53,7 @@ If none of the conditions apply to your cluster, then you can skip to next secti
 
     If the patch fails because the above resource do not exist, you can ignore those errors.
 
-3.  In the `ClusterConfiguration`, if you have set one or more of `noProxy`, `httpProxy`,or `httpsProxy` in `spec.kuberentes.networking` but these values differ from the `values` section of `gatekeeper` addon, then you need to update the gatekeeper addon configuration to match these values. Look up this configmap rendered from `spec.kubernetes.networking`:
+3.  In the `ClusterConfiguration`, if you have set one or more of `noProxy`, `httpProxy`,or `httpsProxy` in `spec.kuberentes.networking` but these values differ from the `values` section of `gatekeeper` addon, then you need to update the Gatekeeper addon configuration to match these values. Look up this ConfigMap rendered from `spec.kubernetes.networking`:
```
[the remark this answers is not in the document]
Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:56]: Ha!

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md (hunk 8 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:26]: @cmastr - the linter stops us from starting a sentence with "there are"

Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:26]: okay - just changed the there are part to the middle, basically: 
ServiceMonitors created using Helm values of the Addon can be adapted in two ways:

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md (hunk 9 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md:27]: I actually looked through this - I think was a misfire from copy and paste from before.

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md (hunk 10 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md:315]: Yeah - I just did some trickeration because the linter doesn't like "as well as" ... I'll do your alternative here.

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md (hunk 8 above)
Comment (cmastr) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md]: dang it!! stupid linter. I like your revision.

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md [hunk 13]
```diff
@@ -102,7 +102,7 @@ If none of the conditions apply to your cluster, then you can skip to next secti
       namespaceSelectorForProxy: {}
     ```
 
-    Copy the values from the `ConfigMap` into gatekeeper `Addon` resource accordingly:
+    Copy the values from the `ConfigMap` into Gatekeeper `Addon` resource accordingly:
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:105]:
```suggestion
    Copy the values from the `ConfigMap` into the Gatekeeper `Addon` resource accordingly:
```

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md [hunk 14]
```diff
@@ -17,13 +17,13 @@ beta: false
...
+Copy or re-create ServiceMonitors using the label `release: kube-prometheus-stack` instead of `release: prometheus-kubeaddons` before processing, using different names or namespaces for the new objects. Or, you can re-label any existing ServiceMonitors before or after processing, but this will introduce an interval where Prometheus is running but the ServiceMonitors have no effect. After the Prometheus-operator in the adapted Prometheus starts, it only looks at ServiceMonitor objects with `release: kube-prometheus-stack` label, and it ignores ServiceMonitors with the old label until they are re-labeled.
 
-There are two ways to adapt ServiceMonitors created using Helm values of the Addon:
+ServiceMonitors created using Helm values of the Addon can be adapted in two ways:
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:26]:
```suggestion
You can adapt the ServiceMonitors created using Helm values of the Addon in two ways:
```

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md [hunk 15]
```diff
@@ -55,29 +55,29 @@ Elasticsearch, elasticsearch-curator, and Kibana Addons are deleted without unin
 
 ### Caveats
 
--   If you customized the Fluent Bit Addon configuration in 1.x, you must add corresponding configOverrides to your 2.x AppDeployment.   The configOverrides should be based on the 2.x defaults combined with your custom modifications.
+-   If you customized the Fluent Bit Addon configuration in 1.x, you must add corresponding configOverrides to your 2.x AppDeployment. The configOverrides should be based on the 2.x defaults combined with your custom modifications.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:58]:
```suggestion
-   If you customized the Fluentbit Addon configuration in 1.x, you must add corresponding configOverrides to your 2.x AppDeployment. The configOverrides should be based on the 2.x defaults combined with your custom modifications.
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md [hunk 16]
```diff
@@ -8,19 +8,23 @@ beta: false
 enterprise: false
 ---
 
+<!-- vale proselint.DateCase = NO -->
+
 Follow these steps only if you are upgrading an AWS cluster.
 
-## Create AMI that is compatible with Konvoy 2.x, Kubernetes v1.20.11, and CentOS 7
+## Create AMI that is compatible with Konvoy 2.x, Kubernetes v1.20.13, and CentOS 7
```
Comment (JohnB64-JBV) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:15]:
```suggestion
## Create an AMI that is compatible with Konvoy 2.x, Kubernetes v1.20.13, and CentOS 7
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md [hunk 17]
```diff
@@ -221,19 +225,19 @@ INFO[2021-11-12T18:22:55-08:00] Created/Updated NVIDIA GPU Feature Discovery Cus
     INFO[2021-11-15T19:59:35-05:00] Run 'export CLUSTER_NAME=konvoy-migration' and follow the rest of the documentation  src="cluster/adopt.go:178"
     ```
 
-1.  Update your environment with the cluster name for use in later steps, by running the shell command from the the last line of output in the previous step:
+1.  Update your environment with the cluster name for use in later steps, by running the shell command from the last line of output in the previous step:
```
Comment (JohnB64-JBV) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:228]:
```suggestion
1.  Run the shell command from the last line of output in the previous step to update your environment with the cluster name, for use in later steps:
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md [hunk 18]
```diff
@@ -456,7 +460,7 @@ You use this AMI to update the cluster Kubernetes version to v1.21.6.
 
 ## Prepare the Dex Addon for Kubernetes v1.21.6
 
-The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as well as with v1.20.11.
+The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as with v1.20.13.
```
Comment (JohnB64-JBV) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:463]:
```suggestion
The Dex Addon acts as the cluster's OpenID Connect identity provider. You must change its configuration so that it works correctly with both Kubernetes v1.21.6 and v1.20.13.
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md [hunk 19]
```diff
@@ -508,7 +512,7 @@ The Dex Addon acts as the cluster's OpenID Connect identity provider. Its config
     export AMI_ID=<your ami ID>
     ```
 
-    ...and then verify that your environment has the AMI ID:
+    And, then verify that your environment has the AMI ID:
```
Comment (JohnB64-JBV) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:515]:
```suggestion
    Then verify that your environment has the AMI ID:
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md [hunk 20]
```diff
@@ -312,7 +312,7 @@ pod "capi-controller-manager-d4b9c7c4c-hkqfl" deleted
 
 ## Prepare the Dex Addon for Kubernetes v1.21.6
 
-The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with Kubernetes v1.21.6, as well as with v1.20.11.
+The Dex Addon acts as the cluster's OpenID Connect identity provider. Its configuration must be changed so that it works correctly with both Kubernetes v1.21.6 and v1.20.11.
```
Comment (JohnB64-JBV) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md:315]:
```suggestion
The Dex Addon acts as the cluster's OpenID Connect identity provider. You must change its configuration so that it works correctly with both Kubernetes v1.21.6 and v1.20.11.
```

**Review by JohnB64-JBV — APPROVED** [review]
My comments are pretty minor, so I approved the PR as is, but it would be great if you could make the changes I indicated before merging.

**Review by gracedo — APPROVED** [review]

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md (hunk 15 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md:58]: Fluent Bit goes by Fluent Bit now - two words. 🤷

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md (hunk 17 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:228]: This is a tough one no matter what. Let me just make two sentences.

Comment (msdolbey) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md:228]:
```suggestion
1.  Update your environment with the cluster name for use in later steps. Do this by running the shell command from the last line of output in the previous step:
```

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md (hunk 4 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md]: updated

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md (hunk 8 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/prepare-apps/index.md]: :)
